### DISCUSSION TITLE
How to delete/remove attachment?

### DISCUSSION THREAD

**Discussion by Snake883** [keeweb/keeweb#2212] [Q&A / Support]
I attached a file to an entry.  How do I delete/remove the attachment/file?

**Comment by youzer-name** [comment]
Click the attachment name, then Control-Delete to remove it.


Longer answer:
In KeeWeb browser interface the uploaded file is shown at the bottom right of the screen:

![image](https://github.com/user-attachments/assets/09fa24cd-229f-4b4e-9e17-be384a51fa6c)

Clicking on the file icon/name brings up the instructions (which are barely legible on my screen)  

![image](https://github.com/user-attachments/assets/ba4cb675-29ce-44a2-8e8d-51fe8796ec6b)
